spec/factories/shift.rb

```diff
@@ -6,8 +6,11 @@
 
     trait :full do
       after(:build) do |shift|
+        day_begin = shift.day.beginning_of_day
+        day_end = shift.day.end_of_day
+
         ShiftEvent::ACTIONS.keys.zip(
-          Array.new(4) { Faker::Time.between(shift.day, shift.day) }.sort
+          Array.new(4) { Faker::Time.between(day_begin, day_end) }.sort
```

Comment:
Since it looks like we're using the same logic here as is in the `:shift_event` factory, can we just leave setting the `occurred_at` out altogether? Or are we relying on the ordering?